### ISSUE TITLE
Update orientation session links in Platform Orientation template

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54065]
## User Story
As a VFS team member, I want to be able to view and follow along with the slides shared during the Platform Orientation sessions so that I can reference again, if needed. 

Assignee: @allison0034 
Peer Reviewer: @briandeconinck 

## Description
Currently, the hyperlinks for the Orientation sessions in the Platform Orientation template link directly to the Calendly sign up page for each specific orientation. This is a problem because VFS teams are not seeing that the additional information, including the slide deck, that is shared on the [Orientation Sessions page](https://depo-platform-documentation.scrollhelp.site/getting-started/calendly-orientation-sessions) on Platform Website. 

The purpose of this ticket is to change every Calendly link in the Platform Orientation template to the Orientation Sessions page anchor link. This will direct VFS teams to Platform Website first, and then the Calendly sign up link.

In addition, the content currently used in the template follows this structure:

- Attend _____ Orientation session (schedule via Calendly)

We likely want to rephrase this content. The checklist item should still be to attend the orientation session, but the part in parentheses should be altered and/or supplemental information added.

## Impacted Artifacts
Platform Orientation template ([orientation-epic.md](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/orientation-epic.md))
Pull request #54862

## Tasks
- [x] Determine what the content should say for each "Attend Orientation Session" checklist item in the ticket. Determine how the anchor link should be embedded in the text.
- [x] Make sure the content for every orientation session checklist item is consistent.
- [x] Changes should be made using a pull request in Github. Be sure to link pr within this ticket.
- [x] Once changes are approved by peer reviewer, merge with master

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] VFS teams are directed to the Orientation Sessions page on Platform Website in order to access the sign-up link for the specified orientation session 
- [x] VFS teams are directed to the slide deck materials

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by allison0034** [comment]
@briandeconinck this is ready for your review. PR #54862.

I played around with what exactly this should say and at first I had the Calendly link and orientation page linked in 2 sentences. Since we just completed research and we know some users felt overwhelmed by this ticket, I decided to trim it down and not link to Calendly directly. Hopefully this will guide users to the orientation page with the slides and they will still be able to easily schedule the session. Let me know if you think this works.
